daemon/logger/splunk/splunk.go

```diff
@@ -22,29 +25,66 @@ import (
 )
 
 const (
-	driverName                  = "splunk"
-	splunkURLKey                = "splunk-url"
-	splunkTokenKey              = "splunk-token"
-	splunkSourceKey             = "splunk-source"
-	splunkSourceTypeKey         = "splunk-sourcetype"
-	splunkIndexKey              = "splunk-index"
-	splunkCAPathKey             = "splunk-capath"
-	splunkCANameKey             = "splunk-caname"
-	splunkInsecureSkipVerifyKey = "splunk-insecureskipverify"
-	splunkFormatKey             = "splunk-format"
-	splunkVerifyConnectionKey   = "splunk-verify-connection"
-	envKey                      = "env"
-	labelsKey                   = "labels"
-	tagKey                      = "tag"
+	driverName                     = "splunk"
+	splunkURLKey                   = "splunk-url"
+	splunkTokenKey                 = "splunk-token"
+	splunkSourceKey                = "splunk-source"
+	splunkSourceTypeKey            = "splunk-sourcetype"
+	splunkIndexKey                 = "splunk-index"
+	splunkCAPathKey                = "splunk-capath"
+	splunkCANameKey                = "splunk-caname"
+	splunkInsecureSkipVerifyKey    = "splunk-insecureskipverify"
+	splunkFormatKey                = "splunk-format"
+	splunkVerifyConnectionKey      = "splunk-verify-connection"
+	splunkGzipCompressionKey       = "splunk-gzip"
+	splunkGzipCompressionLevelKey  = "splunk-gzip-level"
+	envKey                         = "env"
+	labelsKey                      = "labels"
+	tagKey                         = "tag"
 )
 
+const (
+	defaultPostMessagesFrequency       = 5 * time.Second
+	defaultPostMessagesBatchSize       = 1000
+	defaultBufferMaximum               = 10 * defaultPostMessagesBatchSize
+	defaultStreamChannelSize           = 4 * defaultPostMessagesBatchSize
```

Comment:
I think some comments around these 4 constants just to explain how they relate to each other